staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -487,6 +497,11 @@ func (az *Cloud) ensurePublicIPExists(service *v1.Service, pipName string, domai
 	}
 
 	serviceName := getServiceName(service)
+
+	if !needNewPIP {
+		return nil, fmt.Errorf("ensurePublicIPExists for service(%s): pip(%s) doesn't exist", serviceName, pipName)
```

Comment:
fmt.Errorf("PublicIP from annotation azure-pip-name=%s for service %s doesn't exist, ...)

Reply:
The commit has been amended.